.travis.yml

```diff
@@ -1,2 +1,35 @@
 language: r
-cache: packages
\ No newline at end of file
+cache: packages
+
+matrix:
+  include:
+    - os: linux
+      r: release
+      env:
+        - R_CODECOV=true
+    - os: linux
+      r: devel
+    - os: linux
+      r: oldrel
+    - os: osx
+      osx_image: xcode8.3
+
+warnings_are_errors: true
+
+notifications:
+  email:
+    on_success: change
+    on_failure: change
+
+after_success:
+  - if [[ "${R_CODECOV}" ]]; then Rscript -e 'covr::codecov()'; fi
```

Comment:
What do we need to do to set this up? Might as well get the reporting in there straight away?

Reply:
Actually, you're right. We don't need to set up codecov. It will just work:tm: when pushed from travis.

Reply:
Oh cool, didn't realise it just works :crossed_fingers: